Apps/WaitingQueueWeb/Controllers/TicketController.cs

```diff
@@ -48,13 +48,13 @@ public TicketController(IWebTicketDelegate ticketDelegate)
         /// <response code="404">The requested room was not found.</response>
         /// <response code="503">The service is too busy, retry after the amount of time specified in retry-after.</response>
         [HttpPost]
-        [ProducesResponseType(typeof(Ticket), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         [ProducesResponseType(StatusCodes.Status503ServiceUnavailable)]
 
-        public async Task<IActionResult> CreateTicket([FromQuery]string room)
+        public async Task<ActionResult<Ticket>> CreateTicket([FromQuery]string room)
```

Comment:
Nit / comment: The return value could even be changed to just `Task<Ticket>`.

I don't think there's much of a functional difference between the two, except the ActionResult variant doesn't support implicit conversion when the returned payload is an interface, requiring the Ok syntax in that scenario.